test/tools/libtestutils/Common.cpp

```diff
@@ -61,7 +61,9 @@ int dev::test::randomNumber(int _min, int _max)
 
 unsigned short dev::test::randomPortNumber(unsigned short _min, unsigned short _max)
 {
-    return static_cast<unsigned short>(randomNumber(_min, _max));
+    static std::mt19937 randomGenerator(utcTime());
```

Comment:
I believe you should reuse existing RNG, the one that is used by randomNumber().

Reply:
Got it, will make it a global object in an anonymous namespace. Out of curiosity, why do we seed on each call in addition to seeding when we create the rng?

Reply:
After your changes it is seeded only once as it should be. General rule is to seed a PRNG with a good source of entropy so that you will not need to reach out to this source any more later on. Using timestamps for entropy is bad practice, `std::random_device` would be much better, but it can stay for unit testing.

When security is needed, you probably should use secure PRNG which messner twister is not.